shell32.go

```diff
@@ -202,6 +321,7 @@ func init() {
 	shGetPathFromIDList = MustGetProcAddress(libshell32, "SHGetPathFromIDListW")
 	shGetSpecialFolderPath = MustGetProcAddress(libshell32, "SHGetSpecialFolderPathW")
 	shParseDisplayName = MustGetProcAddress(libshell32, "SHParseDisplayName")
+	shGetStockIconInfo = MustGetProcAddress(libshell32, "SHGetStockIconInfo")
```

Comment:
This was also added in Vista. What do you think about adding a new function `MaybeGetProcesAddress`?

Reply:
I'd be ok with that.